src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -146,7 +149,7 @@ const pageQuery = gql`
 			}
 		}
 		contentful {
-			entries(contentType: "page", contentKey: "monthlygood")
+			entries(contentType: "page", contentKey: "monthlygood", preview: true)
```

Comment:
Oops let's get rid of this preview flag please.

Reply:
Removed